packages/cli/src/config/schema.ts

```diff
@@ -316,16 +316,30 @@ export const schema = {
 			env: 'EXECUTIONS_DATA_PRUNE',
 		},
 		pruneDataMaxAge: {
-			doc: 'How old (hours) the finished execution data has to be to get deleted',
+			doc: 'How old (hours) the finished execution data has to be to get soft-deleted',
 			format: Number,
 			default: 336,
 			env: 'EXECUTIONS_DATA_MAX_AGE',
 		},
-		pruneDataInterval: {
```

Comment:
Added by #7439 which was merged prematurely while the conversation was ongoing. Unreleased so it should be fine to modify without marking as a breaking change.

Reply:
It was actually released yesterday in version 1.12.0, according to the last comment on the mentioned PR

Reply:
Sorry. I merged it because it was approved, and it sounded like it was blocking a customer.

Reply:
No worries, I should've marked as a draft.